### PR TITLE
Always scroll iron-list of the overlay when open

### LINKED ISSUE #223: Empty dropdown on iOS after reopen (iron-list v1.2+)

### PR DESCRIPTION
**Pull request by platosha**
Fixes #223

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-combo-box/224)

### CODE REVIEW

File: test/scrolling.html [hunk 1]
```diff
@@ -46,6 +46,60 @@
...
+        });
+
+        it('should be zero when the first item is selected', function() {
+          combobox.value = combobox.items[0];
+          combobox.open();
+
+          expect(combobox.$.overlay.$.scroller.scrollTop).to.equal(0);
+        });
+
+        function expectSelectedItemPositionToBoVisible() {
```
Comment (jouni) [inline test/scrolling.html]: nit: Bo -> Be

**Comment by platosha** [comment]
Review status: 0 of 6 files reviewed at latest revision, 1 unresolved discussion.

---

*[test/scrolling.html, line 74 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/224#-KFdOC428z_17qactZ9O-r1-74:-KFdUE-FRgI56aJ97_T8:-499146009) ([raw file](https://github.com/vaadin/vaadin-combo-box/blob/b4484ebbea6214c9ea0d54d02cef457bce2c69c4/test/scrolling.html#L74)):*
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/224)*

**Comment by tomivirkki** [comment]
Aside the commented-out row :lgtm:

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/224#-:-KFi482OjpPkzyNpEupr:875283375)*

**Comment by tomivirkki** [comment]
Reviewed 4 of 6 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/224)*

**Comment by tomivirkki** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/224)*

**Pull request merged**